src/actions/actionCreators.js

```diff
@@ -1,17 +1,37 @@
 import axios from "axios";
-import * as types from './actionTypes';
+import * as types from "./actionTypes";
 
-export const signup  = (name, email, password) => dispatch => {
+export const signup = userData => dispatch => {
+  const { username, email, password } = userData;
   dispatch({ type: types.SIGNUP_LOAD });
-   axios
-    .post("https://mentor-me-backend.herokuapp.com/signup", { name, email, password })
+  axios
+    .post("https://mentor-me-backend.herokuapp.com/api/auth/register", {
+      username,
+      email,
+      password
+    })
     .then(res => {
-      localStorage.setItem("token", res.data);
-      dispatch({ type: types.SIGNUP_SUCCESS });
+      console.log(res);
```

Comment:
console.log :)

Reply:
Removed them... seem to always forget them